ui/storybook/README.md

```diff
@@ -100,6 +100,32 @@ Storybook configuration lives in `src/config` and includes the following:
 
 Storybook is currently configured to build with [Webpack 5](https://storybook.js.org/docs/react/builders/webpack#webpack-5).
 
+## Visual tests
+
+This project uses a free open source account from [Happo](https://happo.io/) for our visual testing. Visual tests generated for storybook use `happo-plugin-storybook` ([relevant documentation](https://docs.happo.io/docs/storybook)) and are listed under the `perses-storybook` project in Happo. See the `e2e` package for information about visual tests generated using that tooling.
```

Comment:
Configuring these as two projects in happo because it makes it easier to set things up in CI since these tests and the playwright-based tests run very differently.